exonum-java-binding-core/rust/src/storage/core_schema.rs

```diff
@@ -94,3 +94,25 @@ pub extern "system" fn Java_com_exonum_binding_blockchain_CoreSchemaProxy_native
     });
     utils::unwrap_exc_or(&env, res, ptr::null_mut())
 }
+
+/// Returns the configuration for the latest height of the blockchain. Throws
+/// `java.lang.RuntimeException` if the "genesis block" has not been created yet.
```

Comment:
Can we throw custom exception here?  RuntimeException is too general.

Reply:
I agree that it's rather general, but for the moment it's our convention. We can consider it as an area for further improvement.